installation.md

```diff
@@ -0,0 +1,176 @@
+# Installation Instructions (Windows)
+
+The following installation steps should work for a computer with an out-of-the-box Windows OS.
+
+## Required Software
+
+Start by downloading and installing the following applications (with default options, they will be configured in the
+subsequent steps):
+
+- [Anaconda](https://docs.anaconda.com/anaconda/install/windows/)
+- [GitHub Desktop](https://desktop.github.com/)
+- [Intel RealSense Viewer](https://www.intelrealsense.com/sdk-2/) (scroll down to find the download for the viewer)
+- [Visual Studio](https://visualstudio.microsoft.com/downloads/) with the option *Desktop Development with C++* (and
+  make sure *ATL* is part of it)
+- [Motive](https://optitrack.com/support/downloads/motive.html) (including the `.NET` framework)
+- [Visual Studio Code](https://code.visualstudio.com/download) (optionally, suggested editor)
+
+## Get Additional Installation Files
+
+Additional installation and setup files are required to complete the installation steps below. The files can be
+requested from [dominic.reber@epfl.ch](mailto:dominic.reber@epfl.ch) and come as zipped folders.
+
+Unzip *SDK Package.zip*, *Noraxon.Acquire.zip*, and *Chameleon Installer-EPFL1 & 2.zip* somewhere on your computer and
+make sure that the folder structure is as follows:
+
+```bash
+path\to\unzipped\folders
+└──SDK Package
+└──Noraxon.Acquire
+└──Chameleon Installer-EPFL1 & 2
+```
+
+## Acquisition Software Setup Steps
+
+### EMG
+
+1. Get the credit card shaped USB stick from the EMG box and install *Noraxon MR3.10* from the stick.
+
+2. With the explorer, go to `path\to\unzipped\folders\SDK Package` and run `Noraxon.Acquire.1.7.65.0.exe`. Continue with
+   the installation even if Windows complains and leave the default values as is.
+
+3. Turn on the Noraxon Desktop Receiver and plug it to the computer. With the explorer, go
+   to `path\to\unzipped\folders\Noraxon.Acquire\src\easy2\acquirecom` and run `test.exe`. A window should open up that
+   shows three devices, one of which should bee the Noraxon Desktop Receiver. Drag and drop this one to the left, and
+   the other two to the right. A configuration window for the Desktop Receiver will pop up, choose *Detect
+   Configuration* and accept. Confirm and close the windows.
+
+### `surgeon_recording` Library
+
+4. Open GitHub Desktop (you can skip the login steps if you want) and choose the option *Clone a repository from URL*,
+   put [https://github.com/epfl-lasa/surgeon_recording.git](https://github.com/epfl-lasa/surgeon_recording.git) and
+   choose a desired path (for example `C:\Users\USER\Documents\Recordings`), that will here be referred to
+   as `path\to\recordings\`. Therefore, the GitHub repository will have the path `path\to\recordings\surgeon_recording`.
+
+5. Launch Anaconda and open a PowerShell Prompt from the available options. Then.
+   ```bash
+   cd path\to\recordings\surgeon_recording
+   conda create -name surgeon_recording
```

Comment:
you probably need to specify the python version. I think `3.8` is good now, it wasn't before due to `pyrealsense` not available